integrations/server_test.go

```diff
@@ -134,6 +136,275 @@ func TestServer(t *testing.T) {
+					{
+						ID:         1, // This is artificial, but should be reflective of the users actual ID
+						Name:       "billibob",
+						Provider:   "github",
+						Scheme:     "oauth2",
+						SuperAdmin: true,
+						Roles: []chronograf.Role{
+							{
+								Name:         "admin",
+								Organization: "0",
+							},
+						},
+					},
+				},
+			},
+			args: args{
+				server: &server.Server{
+					GithubClientID:     "not empty",
+					GithubClientSecret: "not empty",
+				},
+				method: "POST",
+				path:   "/chronograf/v1/users",
+				payload: &chronograf.User{
+					Name:     "user",
+					Provider: "provider",
+					Scheme:   "oauth2",
+					Roles: []chronograf.Role{
+						{
+							Name:         roles.EditorRoleName,
+							Organization: "0",
+						},
+					},
+				},
+				principal: oauth2.Principal{
+					Organization: "0",
+					Subject:      "billibob",
+					Issuer:       "github",
+				},
+			},
+			wants: wants{
+				statusCode: 201,
+				body: `
+{
+  "links": {
+    "self": "/chronograf/v1/users/2"
+  },
+  "id": "2",
+  "name": "user",
+  "provider": "provider",
+  "scheme": "oauth2",
+  "superAdmin": true,
+  "roles": [
+    {
+      "name": "editor",
+      "organization": "0"
+    }
+  ]
+}`,
+			},
+		},
+		{
+			name:    "POST /users",
+			subName: "Create a New User as a SuperAdmin; SuperAdminNewUsers is false; User on Principal is a SuperAdmin",
+			fields: fields{
+				Config: &chronograf.Config{
+					Auth: chronograf.AuthConfig{
+						SuperAdminNewUsers: false,
+					},
+				},
+				Users: []chronograf.User{
+					{
+						ID:         1, // This is artificial, but should be reflective of the users actual ID
+						Name:       "billibob",
+						Provider:   "github",
+						Scheme:     "oauth2",
+						SuperAdmin: true,
+						Roles: []chronograf.Role{
+							{
+								Name:         "admin",
+								Organization: "0",
+							},
+						},
+					},
+				},
+			},
+			args: args{
+				server: &server.Server{
+					GithubClientID:     "not empty",
+					GithubClientSecret: "not empty",
+				},
+				method: "POST",
+				path:   "/chronograf/v1/users",
+				payload: &chronograf.User{
+					Name:     "user",
+					Provider: "provider",
+					Scheme:   "oauth2",
+					Roles: []chronograf.Role{
+						{
+							Name:         roles.EditorRoleName,
+							Organization: "0",
+						},
+					},
+				},
+				principal: oauth2.Principal{
+					Organization: "0",
+					Subject:      "billibob",
+					Issuer:       "github",
+				},
+			},
+			wants: wants{
+				statusCode: 201,
+				body: `
+{
+  "links": {
+    "self": "/chronograf/v1/users/2"
+  },
+  "id": "2",
+  "name": "user",
+  "provider": "provider",
+  "scheme": "oauth2",
+  "superAdmin": false,
+  "roles": [
+    {
+      "name": "editor",
+      "organization": "0"
+    }
+  ]
+}`,
+			},
+		},
+		{
+			name:    "POST /users",
+			subName: "Create a New User as a SuperAdmin; SuperAdminNewUsers is false; User on Principal is not SuperAdmin",
```

Comment:
what does it mean to create a new user as a superadmin but user not be superadmin on principal? i think this should say `Create a New User as an Admin`

Reply:
it means creating a new user that has super admin status, but the user making the request is not a super admin.

Reply:
I'll fix the wording

Reply:
I see. yeah - clarification would be good